app/templates/main/error.hbs

```diff
@@ -1,7 +0,0 @@
-{{#if slug}}
```

Comment:
I'm afraid we don't have it as SVG. Or at least not somewhere I have access to it. Maybe @ctrlaltjustine knows more about it?